lib/jsdom/living/nodes/Document-impl.js

```diff
@@ -894,6 +895,23 @@ class DocumentImpl extends NodeImpl {
   getSelection() {
     return this._defaultView ? this._defaultView._selection : null;
   }
+
+  // Needed to ensure that the resulting document has the correct prototype chain:
+  _cloneDocument() {
+    const copy = documents.createImpl(
+      this._globalObject,
+      {
+        contentType: this.contentType,
+        encoding: this._encoding,
+        parsingMode: this._parsingMode
+      },
+      Object.getPrototypeOf(this) !== DocumentImpl.prototype
```

Comment:
This has no effect as far as I can tell. It says to don't use the XMLDocument subclass, in a case when we would already not be using it.

Reply:
There’s some cases where browsers create a base `Document` instead of the subclasses.

Reply:
This particular line still has no effect, and removing it causes the same tests to pass before and after.

Reply:
Well, it will become necessary with <https://github.com/jsdom/jsdom/pull/2744>, so I guess I can hold off on introducing it until then.